packages/lib/src/user-query.ts

```diff
@@ -19,17 +25,24 @@ export const askForString = async (message: string, validate?: ValidateFunction,
 		type: 'input',
 		name: 'value',
 		message,
-		validate,
+		validate: validate ? allowEmptyFn(validate) : undefined,
 		default: options?.default,
 	})).value as string
 
 	return value || undefined
 }
 
-export const askForRequiredString = async (message: string, validate?: ValidateFunction, options?: { default: string }): Promise<string> =>
-	askForString(message, (input: string) => input
-		? (validate ? validate(input) : true)
-		: 'value is required', options) as Promise<string>
+export const askForRequiredString = async (message: string, validate?: ValidateFunction, options?: { default: string }): Promise<string> =>  {
```

Comment:
I think we should rename this and `askForString` in another pull request and de-duplicate their now duplicated code.